src/sentry/tasks/store.py

```diff
@@ -169,8 +171,18 @@ def _do_preprocess_event(
             "organization", Organization.objects.get_from_cache(id=project.organization_id)
         )
 
-    platform, symbolication_function = get_symbolication_function(data)
-    if symbolication_function:
+    # Get the list of platforms for which we want to use Symbolicator.
+    # Possible values are `js`, `jvm`, and `native`.
+    # The event will be submitted to Symbolicator for all returned platforms,
+    # one after the other, so we handle mixed stacktraces.
+    stacktraces = find_stacktraces_in_data(data)
+    symbolicate_platforms = get_symbolication_platforms(data, stacktraces)
+    should_symbolicate = len(symbolicate_platforms) > 0
```

Comment:
So apparently, `not symbolicate_platforms` works

```suggestion
    should_symbolicate = not symbolicate_platforms
```

```python
Python 3.11.6 (main, Oct  2 2023, 20:31:07) [Clang 16.0.3 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> item = []
>>> not item
True
```

Reply:
But I need to know if the list is _nonempty_.